examples/simple_example/terraform.tfvars

```diff
@@ -0,0 +1 @@
+../../test/fixtures/shared/terraform.tfvars
```

Comment:
I'm trying to follow why this is symlinked here. Could you expand on that please?

Reply:
You are right, it is not needed, I'll remove it

Reply:
You are right, this symlink is useless, I've removed it.